tasks/deploy-redis.js

```diff
@@ -4,24 +4,21 @@ const os = require('os');
 const util = require('util');
 const gutil = require('gulp-util');
 
-const {
-  env,
-  getRevision,
-  getConfigFor,
-  getRedisClient,
-} = require('./utils');
+const { env, getRevision, getConfigFor, getRedisClient } = require('./utils');
```

Comment:
Can you please remove any churn that does not provide any change?

Reply:
Oh, damn Prettier :(

Reply:
Sorry, I didn't realize that we actually provide a prettier config in the README, maybe 🙏 put it into a prettier config file? :)